server/src/configs/default.json

```diff
@@ -907,7 +907,7 @@
     },
     {
       "name": "GeoURI",
-      "url": "geo:{x},{y}"
+      "url": "geo:{x},{y}?q={x},{y}"
```

Comment:
@TurtIeSocks I'm assuming you can use the replacement variables multiple times in a pattern

Reply:
Not currently but if you change https://github.com/WatWowMap/ReactMap/blob/b309e11ba18b41b91cdaae6d07af02a9d2051423/src/components/popups/Navigation.jsx#L18
to use `replaceAll` instead of `replace` (or a regex) then it shouldn't be a problem.